nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/ops/SDLinalg.java

```diff
@@ -558,4 +560,104 @@ public SDVariable svd(String name, SDVariable input, boolean fullUV, boolean com
     SDVariable out =  new org.nd4j.linalg.api.ops.impl.transforms.custom.Svd(sd,input, fullUV, computeUV, 16).outputVariable();
     return sd.updateVariableNameAndReference(out, name);
   }
+
+  /**
+   * An array with ones at and below the given diagonal and zeros elsewhere.<br>
+   *
+   * @param dataType Data type
+   * @param params  (Size: Range(from=1, to=3))
+   * @return output  (FLOATING_POINT type)
+   */
+  public SDVariable tri(DataType dataType, int... params) {
```

Comment:
Sorry, missed this last time. Let's change the op definition:
Instead of a count=Betweer(1,3) let's split this into 3 integer arguments, like so:
* row
* column
* diagonal - defaultValue = 0